### PR TITLE
[bugfix] update VHACD tutorial script

### PR DESCRIPTION
**Pull request by aclegg3**
## Motivation and Context

Apply recent API changes and add the tutorial to test_examples so future issues will be caught by CI.

## How Has This Been Tested

New CI coverage.

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: tests/test_examples.py [hunk 1]
```diff
@@ -57,6 +57,7 @@ def powerset(iterable):
             "--no-make-video",
         ),
         ("examples/tutorials/semantic_id_tutorial.py", "--no-show-images"),
+        ("examples/tutorials/VHACD_tutorial.py", "--no-show-video", "--no-make-video"),
```
Comment (Skylion007) [inline tests/test_examples.py:60]: Thanks for adding this. Will this error if vhacd is not enabled though? IF so we may need an additional skipif.

**Review by Skylion007 — APPROVED** [review]

File: tests/test_examples.py (hunk 1 above)
Comment (aclegg3) [reply tests/test_examples.py:60]: Yeah, but CI should always hit this with --vhacd currently. 

Considering adding a binding like [this one for bullet in js](https://github.com/facebookresearch/habitat-sim/blob/f568b6564e59269e69f995ad54263b845dac1e22/src/esp/bindings_js/bindings_js.cpp#L127) as suggested by @eundersander.

Comment (Skylion007) [reply tests/test_examples.py:60]: @aclegg3 It already exists: https://github.com/facebookresearch/habitat-sim/blob/f568b6564e59269e69f995ad54263b845dac1e22/habitat_sim/__init__.py#L62

Comment (aclegg3) [reply tests/test_examples.py:60]: Thanks 👍

File: tests/test_examples.py [hunk 2]
```diff
@@ -63,6 +64,16 @@ def test_example_modules(args):
     run_main_subproc(args)
 
 
+@pytest.mark.skipif(
+    not habitat_sim.vhacd_enabled,
+    reason="Requires Habitat-sim to be built with VHACD (--vhacd)",
+)
+def test_vhacd_example():
```
Comment (Skylion007) [inline tests/test_examples.py:71]: Nit can we fully paramterize with pytest.mark.parameterize so that we can add more VHACD tests easily in the future.

**Pull request merged**